src/image_rgb.c

```diff
@@ -160,8 +119,11 @@ static frame *rgb_copy(frame *src, frame *dst)
   dst->width = src->width;
   dst->height = src->height;
   dst->stride = src->stride;
-  dst->data = memalign(ALIGNMENT_BYTES, Rgb_data_size(src));
+  dst->data = malloc(Rgb_data_size(src));
   if (dst->data == NULL) caml_raise_out_of_memory();
+  // Disabling this since we are forgetting about dst->data without freeing:
+  // possible memory leak...
+  assert(0);
```

Comment:
?

Reply:
I was wrong, there is no problem here, removing the assert.